src/WorkOS.net/WorkOS.net.csproj

```diff
@@ -10,7 +10,7 @@
     <IncludeSymbols>true</IncludeSymbols>
     <NeutralLanguage>en-US</NeutralLanguage>
     <PackageId>WorkOS.net</PackageId>
-    <PackageTags>workos;sso</PackageTags>
+    <PackageTags>workos;sso;audit;trail</PackageTags>
```

Comment:
Should this be `audit trail` rather than `audit;trail`?

Reply:
I think standard practice is to keep tags as single words here.